src/main/kotlin/com/atlassian/performance/tools/virtualusers/api/VirtualUserOptions.kt

```diff
@@ -166,6 +169,14 @@ class VirtualUserOptions(
                     .desc("This help")
                     .build()
             )
+            .addOption(
+                Option.builder()
```

Comment:
Just to double-check. Will it work the same way if no results param provided? (AFAIR it should be "test-results" by default).

Reply:
It will work the same way.
`VirtualUserBehavior.results` points to `VirtualUserNodeResult`, which points to `test-results` dir.

- [ ] mention `VirtualUserNodeResult` in `VirtualUserBehavior.results` docs